spec/app_spec.rb

```diff
@@ -885,22 +885,18 @@ def app
     Chef::Cache::KNOWN_PROJECTS.each do |project|
       context "for #{project}" do
         let(:endpoint){ "/stable/#{project}/versions" }
+        let(:params){ { v: "0.0.0" } }
 
         it "exists" do
-          get(endpoint)
+          get(endpoint, params)
           expect(last_response).to be_ok
         end
 
         it "returns the correct JSON data" do
-          get(endpoint)
+          get(endpoint, params)
           expect(last_response.header['Content-Type']).to include 'application/json'
           response = JSON.parse(last_response.body)
-          # Until "automate" is published the omnitruck api needs to translate "automate" to "delivery".
-          if project == 'automate'
-            expect(last_response.body).to match('delivery') unless response.empty?
-          else
-            expect(last_response.body).to match(project) unless response.empty?
-          end
+          expect(last_response.body).to match(project) unless response.empty?
```

Comment:
These specs are functional, so I would need to generate the mock data. I was going to do that as part of the opscode-ci change since this was being coordinated. I suppose no real harm in doing that for now though.